frontend/src/main/components/Users/UsersTable.js

```diff
@@ -111,9 +111,12 @@ export default function UsersTable({ users}) {
     ]
 
     //const columnsToDisplay = showButtons ? buttonColumn : columns;
```

Comment:
Can we remove this one commented out line?  I know it isn't yours, but since it's adjacent to your changes, it woudl be a nice bit of cleanup.

Reply:
Hi, I removed the comment line. Can you check the PR again? Thanks!